src/nummeth.cpp

```diff
@@ -415,6 +414,105 @@ int regula(							// find x given f(x) (regula-falsi method)
 	}
 	return i;
 }			// ::regula
+//-----------------------------------------------------------------------------
+#if 0
```

Comment:
Clean up? What is this all about?

Reply:
Somebody's root finding method that I must have been fooling with.  Deleted.